workspace-manager-clienttests/src/main/resources/serviceaccounts/dev-workspace-app-sa.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "workspace-dev@broad-dsde-dev.iam.gserviceaccount.com",
+  "jsonKeyFilename": "service-account.json",
```

Comment:
Could you make this filename a little more descriptive?

Reply:
Make the workspace manager SA names more descriptive in `render_config.sh` and corresponding json file in `serviceaccounts` directory.